salt/haproxy/init.sls

```diff
@@ -1,6 +1,8 @@
 {% if "admin" not in salt['grains.get']('roles', []) %}
 include:
+  - {{ salt.caasp_cri.cri_salt_state_name() }}
```

Comment:
I don't think we really need this dependency here and, if there is some dependency, it could be reversed: make X include `haproxy` and then add the dependency in X, with `require_in: haproxy`.